services/app-web/src/pages/RateSummary/RateSummary.tsx

```diff
@@ -273,7 +268,7 @@ export const RateSummary = (): React.ReactElement => {
                 </ButtonWithLogging>
             )
         }
-        if (showWithdrawRate && !isWithdrawn) {
+        if (!isWithdrawn) {
```

Comment:
My withdraw orphaned rate work refactored this part of the code. I would like to re-review this logic after the merge conflicts have been resolved.